internal/postgres/reconcile.go

```diff
@@ -168,7 +169,7 @@ func InstancePod(ctx context.Context,
 		// Patroni will set the command and probes.
 
 		Env:       Environment(inCluster),
-		Image:     inCluster.Spec.Image,
+		Image:     config.SetPGContainerImage(inCluster),
```

Comment:
❓ Did we decide the CRD should now say `image` fields are optional? "Required and blank" to trigger this functionality doesn't sound right, so I think we said "yes?"

Reply:
Agree that we had decided to make `image` fields options in support of this change :+1:

Reply:
Agreed, updated.